ecs-init/config/common.go

```diff
@@ -147,3 +147,9 @@ func DockerUnixSocket() (string, bool) {
 func CgroupMountpoint() string {
 	return cgroupMountpoint
 }
+
+// HostCertsDirPath() returns the CA store path on the host (specifically on
+// Amazon Linux platform)
+func HostCertsDirPath() string {
+	return hostCertsDirPath
```

Comment:
This should verify the directory exists even though in the context of Amazon Linux it is fairly benign and seemingly unneeded. Other platforms that aren't suse or ubuntu will be impacted by this change and would otherwise blindly set the certs dir path regardless of its presence.

Reply:
> Other platforms that aren't suse or ubuntu will be impacted by this change and would otherwise blindly set the certs dir path regardless of its presence.

fair. if path doesn't exist, will default to bundle.